components/automate-ui/package.json

```diff
@@ -24,23 +24,23 @@
   },
   "private": true,
   "dependencies": {
```

Comment:
The key changed file; most done by the Angular migration; a few done manually by me.